src/dune_config/dune_config.ml

```diff
@@ -441,7 +458,8 @@ let auto_concurrency =
       in
       loop commands)
 
-let for_scheduler (t : t) stats ~insignificant_changes ~signal_watcher =
+let for_scheduler (t : t) ?watch_exclusions stats ~insignificant_changes
```

Comment:
`?(watch_exclusions = standard_watch_exclusions)` and you can get rid of the Option.value.

Reply:
done